### ISSUE TITLE
Seems to want routes as a function?

### ISSUE THREAD

**Issue by mehcode** [altjs/router#3]
The following works:

``` js
import React from "react"
import {render} from "react-dom"
import createBrowserHistory from "history/lib/createBrowserHistory"
import AltRouter from "alt-router"
import alt from "./alt"
import routes from "./routes"

// TODO: Look into why routes has to be a function that returns itself
var getRoutes = () => routes

render((
  <AltRouter flux={alt} history={createBrowserHistory()} routes={getRoutes} />
), document.getElementById("root"))
```

However if I pass in `routes` directly (as from the README) it errors with:

```
Uncaught TypeError: props.routes is not a function
  AltRouter @   AltRouter.js?5a0a:108
```

**Comment by taion** [comment]
`props.routes` as a function can accept the Alt application instance: https://github.com/altjs/router/blob/v1.1.0/src/AltRouter.js#L63

Not sure if there should be support for `routes` as a non-function e.g. when using singletons though.

**Comment by goatslacker** [comment]
Correct. We can make this work as a non-function

**Comment by goatslacker** [comment]
Published as alt-router@1.1.1

**Comment by goatslacker** [comment]
8a1a9da0f62301a1b30558be70f6a4904c2b1d65
